tests/lib/rules/camelcase.js

```diff
@@ -532,6 +537,30 @@ ruleTester.run("camelcase", rule, {
                 }
             ]
         },
+        {
+            code: "import snake_cased from 'mod'",
```

Comment:
Can we add tests that also explicitly sets the config to `false`?

Reply:
It seems unnecessary. There are also no tests for setting `ignoreDestructuring` option to `false`.

Reply:
I understand that it’s the default value. I think it’s useful as a regression test though.